src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -219,39 +210,19 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
   private final Map<@ClassGetSimpleName String, List<@ClassGetSimpleName String>>
       classToItsUnsolvedInterface = new HashMap<>();
 
-  /**
-   * List of signatures of target methods as specified by users. All signatures have spaces removed
-   * for ease of comparison.
-   */
-  private final Set<String> targetMethodsSignatures;
-
-  /** List of signatures of target fields as specified by users. */
-  private final Set<String> targetFieldsSignatures;
```

Comment:
We could also remove `syntheticReturnTypes` and `syntheticTypes` since their values aren't used anywhere.